src/rdf4cpp/rdf/Literal.hpp

```diff
@@ -40,12 +40,17 @@ class Literal : public Node {
         True = 2,
     };
 
+    /**
+     * @return the effective-boolean-value of this literal as a TriStateBool
+     */
     TriStateBool get_ebv_impl() const;
 
     /**
      * @brief the implementation for all logical, binary operations
      *
-     * @param logic_table the logic table for the binary operation accessed via logic_table[static_cast<size_t>(this->get_ebv_impl())][static_cast<size_t>(other.get_ebv_impl())]
+     * @param logic_table the logic table for the binary operation. It is accessed via
+     *                      logic_table[static_cast<size_t>(this->get_ebv_impl())][static_cast<size_t>(other.get_ebv_impl())].
```

Comment:
any technical reason for the indentation?

Reply:
nope, that was just an accident